updatecli-matrix-targets.yaml

```diff
@@ -9,13 +9,13 @@ matrix:
       pattern:
       image:
       compose_target: &composeNext >-
-        docker-compose/docker-compose.yml
+        docker-compose/pre-release-docker-compose.yml
       compose_key: $.services.alfresco.image
       helm_target: &helmvaluesNext >-
-        helm/alfresco-content-services/values.yaml
+        helm/alfresco-content-services/pre-release_values.yaml
       helm_key: $.alfresco-repository.image.tag
       # keep the unused anchor to detect multiple usages when copying blocks for new versions
-      helm_update_appVersion: &helm_update_appVersion true
+      helm_update_appVersion: &helm_update_appVersion false
```

Comment:
remove it, default is false

put this block to `current` (I can explain better why this is needed in private xD)
```
      # keep the unused anchor to detect multiple usages when copying blocks for new versions
      helm_update_appVersion: &helm_update_appVersion true
```